cli/tests/data/package_build/helloworld/helloworld2.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Yup. We can stop packages from running (old uninstall), but they are still being stored by cosmos.